### ISSUE TITLE
Fix of menu display (icons)

### ISSUE THREAD

**Issue by yaskevich** [jonorthwash/ud-annotatrix#359]
Updating FontAwesome to up-to-date version seems to be needed

**Comment by jonorthwash** [comment]
Could you add a screenshot of what's broken?  (The menu icons look okay for me.)

**Comment by yaskevich** [comment]
Version from repo
![Annotatrix - Chromium72](https://user-images.githubusercontent.com/522902/60203585-a0dd2380-9855-11e9-97fa-42e0a2be9495.png)
Fixed in #356 
![Annotatrix-fixed - Chromium72](https://user-images.githubusercontent.com/522902/60203701-e00b7480-9855-11e9-90c7-0c3ff61674b9.png)

I test in Firefox and Chrome. 
Issues were noticed in Chromium 72.

**Comment by yaskevich** [comment]
Also additional fix for Firefox (specific [CORS policy adaptation](https://stackoverflow.com/questions/20032426/fontawesome-doesnt-display-in-firefox)) was applied, it's important for static version.

**Comment by jonorthwash** [comment]
The version from the repo looks fine in Firefox 60:
![Screenshot from 2019-06-26 23-22-05](https://user-images.githubusercontent.com/963849/60232050-62516400-9869-11e9-951f-3bb42ea204c8.png)

I'd like there to be a little bit less padding on the sides (if possible).

**Comment by yaskevich** [comment]
Reduced paddings: 
![Annotatrix-new-menu](https://user-images.githubusercontent.com/522902/60284088-591cd180-9913-11e9-9860-56ee2b67f53c.png)

**Comment by jonorthwash** [comment]
That looks good, but I'm still confused how it was broken.  It looks fine for me in Firefox (though now broken in Chromium).

**Comment by yaskevich** [comment]
I guess it didn't get broken, it rather was untested with Chrome/Chromium, or this issue occurs only with the newest versions of Blink engine and outdated FontAwesome/Bootstrap.

**Comment by jonorthwash** [comment]
Ah, I see.  So it became a problem when you updated Blink?

**Comment by yaskevich** [comment]
Not exactly, I just opened it in Chromium and spotted the issue. I cannot say when it emerged.
